tests/basecase.py

```diff
@@ -14,18 +14,16 @@ def setUp(self):
             self.setup()
 
 from fakeredis import FakeStrictRedis
-from redbeat import RedBeatSchedulerEntry
-from redbeat.schedulers import add_defaults
+from redbeat.schedulers import RedBeatSchedulerEntry
 
 
 class RedBeatCase(AppCase):
 
     def setup(self):
         self.app.conf.add_defaults({
             'REDBEAT_KEY_PREFIX': 'rb-tests:',
```

Comment:
Should REDBEAT_KEY_PREFIX be removed or a comment included explaining why we appear to double set this value?